### PR TITLE
Add recipe for XDesign

### PR DESCRIPTION
**Pull request by carterbox**
Adds a build recipe for XDesign, a Python package for creating x-ray imaging phantoms, simulating data acquisition, and benchmarking x-ray tomographic image reconstruction.

### CODE REVIEW

**Comment by carterbox** [comment]
@conda-forge/help-python

Hints on why the [one Azure windows build is failing ](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=17906)? This recipe is supposed to be `noarch` and `python>=3`, so I don't understand why it would fail when the others are passing.

File: recipes/xdesign/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b0779b47c93f29d28b20f68355065c0bb7b470fa990898e62c6571078ad266d7
+
+build:
+  noarch: python
+  number: 2
```
Comment (chrisburr) [inline recipes/xdesign/meta.yaml]: This should start from 0 for new versions and only be bumped when changing an already deployed (to anaconda.org) version:
```suggestion
  number: 0
```

File: recipes/xdesign/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{% set name = "xdesign" %}
+{% set version = "v0.5.1" %}
```
Comment (chrisburr) [inline recipes/xdesign/meta.yaml]: I would remove the `v` from here:
```suggestion
{% set version = "0.5.1" %}
```

File: recipes/xdesign/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/xdesign/meta.yaml]: and only put it here:
```suggestion
  url: https://github.com/tomography/{{ name }}/archive/v{{ version }}.tar.gz
```

File: recipes/xdesign/meta.yaml (hunk 2 above)
Comment (carterbox) [reply recipes/xdesign/meta.yaml]: I assume this is because you don't want the `v` to be included in the package version string on conda?

Comment (chrisburr) [reply recipes/xdesign/meta.yaml]: Yeah. The version comparison is quite robust and can handle it but it's nice to keep strings clean where practical.

**Pull request merged**